include/deal.II/base/utilities.h

```diff
@@ -1030,27 +1030,15 @@ namespace Utilities
 {
   template <int N, typename T>
   inline T
-  fixed_power(const T n)
+  fixed_power(const T x)
   {
-    Assert(N >= 0, ExcNotImplemented());
```

Comment:
I think it would be useful to leave this in if `T` is an integral type. There's a type traits in `std::` for this somewhere.